docs/install/install-server/install-server.inc

```diff
@@ -59,36 +59,91 @@ Install the software
         The first package (inmanta-oss) contains all the code and the commands. The server and the agent packages install config
         files and systemd unit files. The dashboard is installed with the server package.
 
-    .. tab:: Other Linux and Mac
+
+    .. tab:: Debian, Ubuntu and derivatives. 
 
         First make sure Python >= 3.6 and git are installed. Inmanta requires many dependencies so it is recommended to create a virtual env.
         Next install inmanta with pip install in the newly created virtual env.
 
+        Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
+
         .. code-block:: sh
 
             # Install python3 >= 3.6 and git
             sudo python3 -m venv /opt/inmanta
             sudo /opt/inmanta/bin/pip install inmanta
             sudo /opt/inmanta/bin/inmanta --help
+            
+            # Install PostgreSQL
+            sudo apt-get install postgresql postgresql-client
```

Comment:
The installation instructions are incomplete. Some issues I found on ubuntu:
* `python3-venv` should be installed in order to run `python3 -m venv`
* `wheel` should be install in the venv to make `/opt/inmanta/bin/pip install inmanta` succeed
* `gcc` should be installed in order to compile some dependencies of the inmanta package.

I am wondering in how much detail we should go here. The `Debian, Ubuntu and derivatives` section is still very generic. So we can never be sure that everything will work on each distribution. But some crucial thing we should definitively mention, such as the dependencies on `gcc` and `wheel`. These dependencies exist for all platforms.

Reply:
This part was there since the beginning, I believe adding those steps would not cause any issues.

Reply:
`gcc` and `wheel` are also installed by default on Ubuntu derivatives. I have never had to install them manually myself. Correct me if I'm wrong

Reply:
I checked again: if you use a cloud image these are minimal systems and they do not contain gcc

Reply:
ok. I will add them too